sdk/src/account_meta.rs

```diff
@@ -0,0 +1,133 @@
+//! Types used
+
+use anchor_lang::{AnchorDeserialize, AnchorSerialize};
+use solana_program::pubkey::Pubkey;
+
+use crate::{
+    compressed_account::CompressedAccountWithMerkleContext,
+    error::LightSdkError,
+    merkle_context::{
+        pack_address_merkle_context, pack_merkle_context, AddressMerkleContext,
+        PackedAddressMerkleContext, PackedMerkleContext, RemainingAccounts,
+    },
+};
+
+#[derive(Debug, Clone, AnchorSerialize, AnchorDeserialize, PartialEq, Default)]
+pub struct LightAccountMeta {
+    /// Lamports.
+    pub lamports: Option<u64>,
+    /// Address of the account (the address can change).
+    pub address: Option<[u8; 32]>,
+    /// Data of the account.
+    pub data: Option<Vec<u8>>,
+    /// Size of the account.
+    ///
+    /// Relevant only for `init` accounts.
+    ///
+    /// This option is not mandatory, but it's recommended and used for
+    /// memory allocation of the new account data in the program. Not
+    /// setting it might result in wasteful re-allocations.
+    pub space: Option<u16>,
```

Comment:
Whats the use case to pass space as instruction data?
(anchor usually defines account size in program)

Reply:
True, we should probably just let developers define the space on the program side.